modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -109,6 +109,13 @@ The bootstrap repository must be created before bootstrapping the client with {w
 For more information, see xref:client-configuration:bootstrap-repository.adoc#_prepare_to_create_a_bootstrap_repository[Prepare to Create a Bootstrap Repository]
 ====
 
+[IMPORTANT]
+====
+Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute Salt commands on the client with the bundled Python.
```

Comment:
```suggestion
Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle and to execute Salt commands on the client with the bundled Python.
```

Reply:
@0rnela  `to` was used in meaning of `into`, but not the way you suggested.
Maybe not really clear, but I mean the Salt Bundle is deployed `to` [path]``/var/tmp``... But now I'm not sure how which one is more correct.

Reply:
We probably need a native speaker to read check this in `master` first.  Joseph approved this.  Thus I think the current wording isn't completely wrong.